WordPress/Sniffs/VIP/SuppressFiltersSniff.php

```diff
@@ -8,61 +8,203 @@
  */
 
 /**
- * Suppress Filters need to be 'true' when getting posts using get_posts.
+ * Support_filters must be false when using get_posts(), wp_get_recent_posts(), get_children().
```

Comment:
@GaryJones thanks for your concern. yes, it's still incomplete, I need some guidance on this, like what it should be?